CONTRIBUTING.rst

```diff
@@ -180,16 +213,71 @@ To run a subset of tests::
 
 $ pytest tests.test_xscen
 
+To run specific code style checks::
+
+    $ black --check xscen tests
+    $ isort --check xscen tests
+    $ blackdoc --check xscen docs
+    $ ruff xscen tests
+    $ flake8 xscen tests
+
+To get ``black``, ``isort ``blackdoc``, ``ruff``, and ``flake8`` (with plugins ``flake8-alphabetize`` and ``flake8-rst-docstrings``) simply `$ pip install` them into your environment.
+
```

Comment:
```suggestion
To get ``black``, ``isort ``blackdoc``, ``ruff``, and ``flake8`` (with plugins ``flake8-alphabetize`` and ``flake8-rst-docstrings``) simply `$ mamba install` them into your environment.

```
No ?

Reply:
I mentioned pip here since `flake8-alphabetize` is the only one not on `conda` at the moment. (I could add it, I suppose). These will all work fine installed with `conda` or `pip` in an Anaconda environment.

Reply:
Opting for "simply install them with `conda` or `pip` into your environment". Changes will come from cookiecutter.